.readme-partials.yml

```diff
@@ -0,0 +1,5 @@
+body: |-
+        ### Using TypeScript
+
+        `@google-cloud/cloudbuild` provides TypeScript type definitions.
```

Comment:
Heh, that doesn't look right :) Should we use a template variable here?

Reply:
fixed :ok_hand: